crypto/signer.go

```diff
@@ -17,47 +17,20 @@ package crypto
 
 import (
 	"crypto"
-	"crypto/rand"
-
-	"golang.org/x/crypto/ed25519"
 )
```

Comment:
Can squash this into 1 line. (my favourite nit :D)

Reply:
Done. 😃